pkgs/development/python-modules/pystac-client/default.nix

```diff
@@ -0,0 +1,84 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  pytestCheckHook,
+  pythonOlder,
+
+  pystac,
+  pytest-benchmark,
+  pytest-console-scripts,
+  pytest-mock,
+  pytest-recording,
+  python-dateutil,
+  requests,
+  requests-mock,
+  setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "pystac-client";
+  version = "0.8.3";
+  pyproject = true;
+  disabled = pythonOlder "3.9";
+
+  src = fetchFromGitHub {
+    owner = "stac-utils";
+    repo = "pystac-client";
+    rev = "v${version}";
+    hash = "sha256-tzfpvNtj+KkKjA75En+OwxYQWGzxHLACLkzWT2j/ThU=";
+  };
+
+  build-system = [ setuptools ];
+
+  propagatedBuildInputs = [
+    pystac
+    python-dateutil
+    requests
+  ];
+
+  nativeCheckInputs = [
+    pytest-benchmark
+    pytestCheckHook
+    pytest-console-scripts
+    pytest-mock
+    pytest-recording
+    requests-mock
+  ];
+
+  disabledTests = [
+    # tests are accessing network
```

Comment:
Is it possible to filter for tests marked `vcr`, that is, decorated with `@pytest.mark.vcr`? That should simplify the list, and clarify whether any tests are missing a marker to indicate that they need Internet access to work.

Reply:
Good idea. Done now.